aws-glue/jobs/reporting-ratio_stats.py

```diff
@@ -12,80 +34,69 @@
 job = Job(glueContext)
 
 # Import necessary libraries
-import boto3
-from statsmodels.distributions.empirical_distribution import ECDF
-import io
-import math
-import numpy as np
-import pandas as pd
-import re
-import s3fs
-import time
-import statsmodels.api as sm
-import warnings
+
 
 # Define AWS boto3 clients
-athena_client = boto3.client('athena')
-glue_client = boto3.client('glue', region_name='us-east-1')
-s3_client = boto3.client('s3')
+athena_client = boto3.client("athena")
+glue_client = boto3.client("glue", region_name="us-east-1")
+s3_client = boto3.client("s3")
 
 # Define s3 and Athena paths
-athena_db = 'iasworld'
-
-s3_bucket = 'ccao-data-warehouse-us-east-1'
-s3_prefix = 'reporting/ratio_stats/'
-s3_output = 's3://'+ s3_bucket + '/' + s3_prefix
-s3_ratio_stats = 's3://'+ s3_bucket + '/' + s3_prefix + 'ratio_stats.parquet'
+athena_db = "iasworld"
+
+s3_bucket = "ccao-data-warehouse-us-east-1"
+s3_prefix = "reporting/ratio_stats/"
+s3_output = "s3://" + s3_bucket + "/" + s3_prefix
+s3_ratio_stats = "s3://" + s3_bucket + "/" + s3_prefix + "ratio_stats.parquet"
+s3_ratio_stats_test = (
+    "s3://" + s3_bucket + "/" + "reporting/ratio_stats_test/" + "ratio_stats.parquet"
+)
```

Comment:
**[Nitpick]** In the PR body, you mentioned that the outputs should be reset to the prod folder:

> The outputs are still set to the test folder. It needs to be shifted back to the original folder.

I agree with this, but I think that now that we're proposing an official change to the script via this pull request, there's no need for the test bucket anymore, so we might as well return to using `s3_ratio_stats` now:

```suggestion
```

If we need to test any further changes, we can always switch it back!